examples/run_CPMG.py

```diff
@@ -31,31 +30,27 @@
 # {{{set phase cycling
 phase_cycling = True
 if phase_cycling:
-    ph1_cyc = r_[0, 1, 2, 3]
-    nPhaseSteps = 4
+    ph1_cyc = r_[0, 2, 1, 3, 2, 4, 3, 5]
```

Comment:
we want this:
```
In [59]: for k in range(4):
    ...:     for j in range(2):
    ...:         print("90 pulse phase is",(j*2+k)%4,"180 pulse phase is",(k+1)%4) # modulus, converts 4 to 0
    ...: 
90 pulse phase is 0 180 pulse phase is 1
90 pulse phase is 2 180 pulse phase is 1
90 pulse phase is 1 180 pulse phase is 2
90 pulse phase is 3 180 pulse phase is 2
90 pulse phase is 2 180 pulse phase is 3
90 pulse phase is 0 180 pulse phase is 3
90 pulse phase is 3 180 pulse phase is 0
90 pulse phase is 1 180 pulse phase is 0
```

Reply:
this can be achieved with list comprehensions like so:
```
ph1 = np.array([(j*2+k)%4  for k in range (4) for j in range(2)]) # inner loop comes last
ph2 = np.array([(k+1)%4  for k in range (4) for j in range(2)]) # inner loop comes last
```

Reply:
done